app/src/main/java/com/aacademy/homework/worker/MovieUpdateWorkerRepository.kt

```diff
@@ -0,0 +1,22 @@
+package com.aacademy.homework.worker
+
+import androidx.work.Constraints
+import androidx.work.NetworkType
+import androidx.work.PeriodicWorkRequestBuilder
+import com.aacademy.homework.BuildConfig
+import java.util.concurrent.TimeUnit
+
+object MovieUpdateWorkerRepository {
+
+    const val WORK_NAME = "${BuildConfig.APPLICATION_ID}.MovieUpdateWorker"
+
+    private val constraints =
+        Constraints.Builder()
+            .setRequiresCharging(true)
+            .setRequiredNetworkType(NetworkType.CONNECTED)
```

Comment:
The assignment says - "when he connected to unmetered network"  ©NikitaVladimirovich